base/linalg/cholesky.jl

```diff
@@ -101,16 +101,14 @@ function _chol!(A::AbstractMatrix, ::Type{LowerTriangular})
             end
             A[k,k] = Akk
             AkkInv = inv(Akk)
-            for j = 1:k
+            for j = 1:k - 1
                 for i = k + 1:n
```

Comment:
`@simd` here makes the function 1.5x faster for large matrices. (Same could be done for UpperTriangular)?